src/main/java/wolox/gmjavatraining/GmJavaTrainingApplication.java

```diff
@@ -3,9 +3,7 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
-import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
-@EnableJpaRepositories("wolox.gmjavatraining.persistence.repo")
 @EntityScan("wolox.gmjavatraining.persistence.model")
```

Comment:
This one also needs to be removed.

Reply:
Hum ok, I did not understand that. I will do it!